storybook-utilities/markdownLinkBuilder.js

```diff
@@ -0,0 +1,7 @@
+const markdownLinkBuilder = (component) => {
```

Comment:
I think you should call this like markdownDocumentationLinkBuilder or something, i think markdownLink is a little vague.